src/jrd/PreparedStatement.cpp

```diff
@@ -109,6 +109,11 @@ namespace
 				item.length = sizeof(Decimal128);
 				break;
 
+			case dtype_dec_fixed:
```

Comment:
@AlexPeshkoff I think the scale should be set here too.

Reply:
Thank you, fixed all suggested places